### ISSUE TITLE
Direct link to GoCardless API docs in footer?

### ISSUE THREAD

**Issue by jamesshedden** [gocardless/splash-pages#393]
^^^ That might be useful?

As suggested by @jamescun.

**Comment by jamescun** [comment]
Yes I should have created an issue instead of bothering sheds.

Basically it has become muscle memory for me (and I'm sure others) that docs will be linked in the footer (although occasionally header for FOSS projects). I did this on the homepage, assuming that `API` linked to the docs but instead it took me to the feature page.

**Comment by samjewell** [comment]
I think that feature page outlining the Pro API is now not linked from anywhere though :-(

**Comment by jamescun** [comment]
I was just thinking from a developers point of view, i.e. "I want the API docs, not sales pages"

**Comment by samjewell** [comment]
Yup no probs, happy to keep this change, just wondering whether we can link to that feature page somewhere else instead, so that it at least gets crawled by Google for example...

**Comment by angusbayley** [comment]
Think you might have that the wrong way round Sam. There's no marketing page for the pro API except for the pro page. There is one for the regular API, and yeah it is linked from the features page too so no worries about crawling (bottom right here):

![screen shot 2015-01-26 at 11 30 47](https://cloud.githubusercontent.com/assets/5781410/5899092/cff89982-a54e-11e4-85f9-c44cb56b951f.png)

**Comment by samjewell** [comment]
Ah thanks a lot Angus, sorry about that guys
